### DISCUSSION TITLE
Automatismi con stati solari

### DISCUSSION THREAD

**Discussion by zolabus** [indomus/forum#1706] [HUB: Home Assistant]
Ciao a tutti vorrei che le luci del mio corridoio si accendessero al mio ingresso in casa solo dopo il tramonto e prima dell'alba
L'automazione così fatta però non funziona mentre funziona perfettamente impostando degli orari (che però dovrei modificare ogni stagione). Devo per caso "attivare" in qualche modo gli stati solari?
Grazie mille

**Comment by Moxser** [comment]
Io ti consiglierei ti utilizzare come condizione se il sole è sopra o sotto l'orizzonte. Anche io ho avuto dei problemi con Alba e tramonto.
